module-2/lab-mongo/Untitled3.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
- "All the companies that have a null type of value on the category_code field". Null is not the same as None. I would query both separately. I encourage you to ask Marc the difference between Null and undefined in JavaScript. It could be interesting. 
- Remember to ask Marc to solve question number 12!!
- I invite you to explore the aggregate () function in mongo, similar to groupby. Play with this data set to get some aggregations on data stored in MongoDB.